src/ResourceManager/DevSpaces/Commands.DevSpaces/AksSdk/ContainerServiceClient.cs

```diff
@@ -0,0 +1,317 @@
+// <auto-generated>
```

Comment:
Why is the AKS SDK part of the PR?

Id DevSpaces a subset of AKS? 

AKS already has the generated SDK as a [part of our repo](https://github.com/Azure/azure-powershell/tree/preview/src/ResourceManager/Aks/Commands.Aks/Generated).

Reply:
@jikuma in that case I would recommend moveing the generated bits under the [AKS folder](https://github.com/Azure/azure-powershell/tree/preview/src/ResourceManager/Aks/Commands.Aks/Generated) to be under the [ResourceManager.common](https://github.com/Azure/azure-powershell/tree/preview/src/ResourceManager/Common) and comsuming the same form both AKS and DevSpaces

Reply:
Thanks @praries880 , I am making the changes. I have one question though, I think we can refer azure sdk from AKS folder itself in DevSpaces project. I don't want to change AKS project is that ok with you?

Reply:
@jikuma Having the SDK code reside under the AKS folder can have unintended consequences for your RP, if the SDK gets updated you might get impacted. 

Thinking about it, i suggest you follow what we did for the  [Network's SDK](https://github.com/Azure/azure-powershell/tree/preview/src/Common/Commands.Common.Network).
or 
[Compute SDK](https://github.com/Azure/azure-powershell/tree/preview/src/Common/Commands.Common.Compute).

That way different RPs can depend on different versions of the SDK and not break (this also makes sure thigs work correctly in netcore). I would suggest following a similar pattern with the AKS SDK, where you check in the AKS SDK version you depend as a specific version (look at how namespaces are named under the different versions checked in)... This way you dont have to modify AKS's code and you dont get tied to the version AKS consumes.

Reply:
Moved to nuget